### ISSUE TITLE
Implement torch.nn.functional.normalize

### ISSUE THREAD

**Issue by athitten** [Lightning-AI/lightning-thunder#334]
### 🚀 Feature

Implement [`torch.nn.functional.normalize`](https://pytorch.org/docs/stable/generated/torch.nn.functional.normalize.html)

### Motivation

NeMo's [`MegatronClipModel`](https://github.com/NVIDIA/NeMo/blob/main/examples/multimodal/vision_language_foundation/clip/megatron_clip_pretrain.py)

cc @tfogal
